frontend/src/container/GridCardLayout/GridCardLayout.tsx

```diff
@@ -118,7 +118,11 @@ function GraphLayout({ onAddPanelHandler }: GraphLayoutProps): JSX.Element {
 					)}
 
 					{addPanelPermission && (
-						<Button onClick={onAddPanelHandler} icon={<PlusOutlined />}>
+						<Button
+							onClick={onAddPanelHandler}
+							icon={<PlusOutlined />}
+							data-testid="add-panel"
```

Comment:
Can we do this for all the `data-testid` in the PR so that there will be only a single source of truth?
```suggestion
							data-testid={addPanelID}
```

Reply:
@Rajat-Dabade ideally one data-testid should be linked to only one element , so basically there is always only one source of truth.

Reply:
I was concerned about variables not with respect to `data-testid`. 

For example, the test case will fail if I change the string "add-panel" to something else. If I create a variable and use it at both places and change it at any time, the test case will still pass as the `data-testid` for the component and the test case will be the same. 

Hope it is clear, let me know.

Reply:
https://signoz-team.slack.com/archives/C062T9FF9C3/p1700202040468499?thread_ts=1700165038.498609&cid=C062T9FF9C3